BUILD/phylums/banish.dat

```diff
@@ -0,0 +1,11 @@
+dude	loc:The Black Forest
+dude	loc:Twin Peak
+dude	loc:Whitey's Grove
+beast	loc:The Hidden Park
+beast	loc:Inside the Palindome
+beast	loc:The Penultimate Fantasy Airship
+undead	loc:The Haunted Library
+undead	loc:The Red Zeppelin
```

Comment:
should we banish dudes in the red zeppelin? Gives up on getting glark cables but then always get skeleton which is quest progress. Basing it on this loathers article
https://www.loathers.net/analysis/2023-06-peagle/

Reply:
I think that could be an issue with typically going right into Palindome after that. BUT, in theory we could hit enough combats to reset the counter and then banish beasts inside the Palindome to change the banished phylum. I'm thinking of adding a block into that and other zones like the Cyrpt that we absolutely need a specific phylum to skip that zone until we can actually do stuff in it (whether it is a new phylum banish or just straight up adventuring in it because there is no current relevant phylum banished)